types/index.d.ts

```diff
@@ -0,0 +1,117 @@
+interface LocalNotificationAction {
+    id: string | null;
+    type?: 'button' | 'input';
+    title?: string | null;
+    launch?: boolean;
+    ui?: boolean;
+    needsAuth?: boolean;
+    icon?: string | null;
+    emptyText?: string;
+    submitTitle?: boolean;
+    editable: boolean;
+    choices?: string[];
+    defaultValue?: string;
+}
+
+interface LocalNotification {
+    actionGroupId?:  number | string | null;
+    actions?: LocalNotificationAction[];
+    attachments?: string[];
+    autoClear?: boolean;
+    badge?: number | null;
+    channel?: string | null;
+    color?: string | null;
+    data?: any;
+    defaults?: number;
+    foreground?: boolean;
+    group?: string | null;
+    groupSummary?: boolean;
+    icon?: string | null;
+    id?: number;
+    launch?: boolean;
+    led?: boolean | string | [string, number?, number?] | {
+        on?: number;
+        off?: number;
+    };
+    lockscreen?: boolean;
+    mediaSession?: string | null;
+    number?: number;
+    priority?: number;
+    progressBar?: boolean | {
+        enabled?: boolean;
+        value?: number;
+        maxValue?: number;
+        intermediate?: boolean;
+    };
+    showWhen?: boolean;
+    silent?: boolean;
+    smallIcon?: string;
+    sound?: boolean | string;
+    sticky?: boolean;
+    summary?: string | null;
+    text?: string;
+    title?: string;
+    trigger?: {
```

Comment:
When using trigger like this

```
cordova.plugins.notification.local.schedule({
    title: 'Design team meeting',
    trigger: { in: 1, unit: 'hour' }
});
```

The "at" and "every" property should be optional